scrips/eventTile.js

```diff
@@ -0,0 +1,83 @@
+import { createDomElement, appendChildren, displayTime } from "./utils.js";
+import { cellHeight, currentEventTileId } from "./calendarVars.js";
+import { openModal } from "./modal.js";
+import { constructNewEvent, getEventLength } from "./eventsData.js";
+import { currentEventDataKey, storeDataInLocalStorage } from "./handleLocalStorage.js";
+import { minutesToHour } from "./timeCalculations.js";
+
+const calculateTopPosition = (eventData) => {
+    const startHour = parseInt(eventData.startTime.hour, 10);
+    const startMin = minutesToHour(eventData.startTime.minutes);
+
+    return (startHour + startMin) * cellHeight;
+};
+
+const styleEventTile = (eventTile, eventData) => {
+    const eventLength = getEventLength(eventData);
+
+    eventTile.style.top = `${calculateTopPosition(eventData)}px`;
+    eventTile.style.height = `${eventLength * cellHeight - 4}px`;
```

Comment:
I assume that `4` here is a padding/margin top of the cell?

```suggestion
    eventTile.style.height = `${eventLength * cellHeight - cellMarginTop}px`;
```